sdk/typescript/src/signers/txn-data-serializers/call-arg-serializer.ts

```diff
@@ -28,6 +28,14 @@ export class CallArgSerializer {
   async extractObjectIds(txn: MoveCallTransaction): Promise<ObjectId[]> {
     const args = await this.serializeMoveCallArguments(txn);
     return args
+      .map((arg) =>
```

Comment:
what about string vec?

Reply:
Here's the definition: https://github.com/MystenLabs/sui/blob/abc23b9d30ddd5eb27d8d7db371bf2035f67335b/sdk/typescript/src/types/sui-bcs.ts#L199. We only count Object and Object vec as input objects